src/ontology/hp-edit.owl

```diff
@@ -15043,7 +15038,6 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://purl.obolibrary.org/obo/RO_0000052> "inheres_in"^^xsd:string)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#shorthand> <http://purl.obolibrary.org/obo/RO_0000052> "inheres_in"^^xsd:string)
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/RO_0000052> "inheres in"^^xsd:string)
-SubObjectPropertyOf(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/RO_0002314>)
```

Comment:
why are we deleting this? We added this intentionally. Is this now somewhere else or is it not important anymore? @cmungall @matentzn

Reply:
The odk makes sure that all axioms that are not HP native are pulled in from the outside automatically. This one is being pulled from RO!

Reply:
Thanks for the clarification